### ISSUE TITLE
Grav Content Array - Cannot enter categories with spaces

### ISSUE THREAD

**Issue by blizam** [gantry/gantry5#2353]
If you have a category in Grav named 'more than one word' -- which is also how Grav stores it in YAML -- you can't enter that category name, with spaces, in the Grav Content Particle as the input.selectize field interprets spaces as delimiters...

is there a character I can use instead of a space to still target a Grav category that has space(s)?

**Comment by blizam** [comment]
It's useful if you want to output the Category name somewhere for sure, to have initial caps proper category titles... I sort of assume this behaves the same way in Joomla and WP... having the field be a prepopulated multi-select dropdown of available categories would be nice :)

**Comment by simmonsr** [comment]
I would just use a hyphen - in this case. `more-than-one-word`

**Comment by blizam** [comment]
Yeah, but then if you want to output your tag or category name on a readable way you'd have to similarly strip the hyphens
